src/languages/javascript.js

```diff
@@ -325,6 +325,23 @@ export default function(hljs) {
         ],
         illegal: /%/
       },
+      {
+        className: 'function',
+        // we have to count the parens to make sure we actually have the
+        // correct bounding ( ) before the =>.  There could be any number of
+        // sub-expressions inside also surrounded by parens.
+        begin: hljs.UNDERSCORE_IDENT_RE + '(\\([^(]*' +
+              '(\\([^(]*' +
+                '(\\([^(]*' +
+                '\\))?' +
+              '\\))?' +
+            '\\))\\s*{',
```

Comment:
Can you give me an example of nested parens here?

Reply:
Huh?  ((3))

Reply:
And what is the meaning of it as a function parameter?
Okay, I really just want to say that the regex doesn't handle `((3) )`, which I believe to be correct.

Reply:
Example: 

```
  other(a = (3+2)) {
    console.log(a)
  }
```

I suppose we can eat spaces on both sides, though that's unusual. :-)